cmd/spire-server/cli/run/run_test.go

```diff
@@ -467,6 +467,17 @@ func TestNewServerConfig(t *testing.T) {
 				require.Nil(t, c)
 			},
 		},
+		{
+			msg:         "invalid bind_port should return an error",
```

Comment:
can you add a test case related with created issue? to verify that we are now supporting a hostname